config/common.php

```diff
@@ -39,7 +39,7 @@
         $trackedServices = (array)($params['debugger.trackedServices'] ?? []);
         $decoratedServices = (array)($params['container.decorators'] ?? []);
         $path = $container->get(Aliases::class)->get("@runtime/cache/container-proxy");
-        $logLevel = ContainerProxy::LOG_ARGUMENTS | ContainerProxy::LOG_RESULT | ContainerProxy::LOG_ERROR;
+        $logLevel = $params['debugger.logLevel'] ?? null;
```

Comment:
`logLevel` can't be null
https://github.com/yiiliveext/yii-debug/blob/update-debug/src/Proxy/ContainerProxyConfig.php#L28

Reply:
Done